src/pages/organize/[orgId]/campaigns/[campId]/surveys/[surveyId]/index.tsx

```diff
@@ -27,9 +34,59 @@ interface SurveyPageProps {
   surveyId: string;
 }
 
-const SurveyPage: PageWithLayout<SurveyPageProps> = () => {
-  return <></>;
+const SurveyPage: PageWithLayout<SurveyPageProps> = ({
+  campId,
+  orgId,
+  surveyId,
+}) => {
+  const model = useModel(
+    (env) => new SurveyDataModel(env, parseInt(orgId), parseInt(surveyId))
+  );
+  const onServer = useServerSide();
+
+  if (onServer) {
+    return null;
+  }
+
+  const { data: survey } = model.getData();
+
+  if (!survey) {
+    return null;
+  }
+
+  return (
+    <Box
+      alignItems="center"
+      display="flex"
+      justifyContent="center"
+      paddingTop={8}
+    >
+      {survey.elements.length ? (
+        <></>
+      ) : (
```

Comment:
Maybe this doesn't need to be a ternary, but could just use the `&&` pattern instead?